andino_navigation/launch/andino_simulation_navigation_launch.py

```diff
@@ -0,0 +1,249 @@
+    )
+
+    declare_slam_cmd = DeclareLaunchArgument(
+        'slam', default_value='False', description='Whether run a SLAM'
+    )
+
+    declare_map_yaml_cmd = DeclareLaunchArgument(
+        'map',
+        default_value=os.path.join(
+            andino_navigation_dir, 'maps', 'turtlebot3_world.yaml'
+        ),
+        description='Full path to map file to load',
+    )
+
+    declare_use_sim_time_cmd = DeclareLaunchArgument(
+        'use_sim_time',
+        default_value='true',
+        description='Use simulation (Gazebo) clock if true',
+    )
+
+    declare_params_file_cmd = DeclareLaunchArgument(
+        'params_file',
+        default_value=os.path.join(andino_navigation_dir, 'params', 'nav2_params.yaml'),
+        description='Full path to the ROS2 parameters file to use for all launched nodes',
+    )
+
+    declare_autostart_cmd = DeclareLaunchArgument(
+        'autostart',
+        default_value='true',
+        description='Automatically startup the nav2 stack',
+    )
+
+    declare_use_composition_cmd = DeclareLaunchArgument(
+        'use_composition',
+        default_value='True',
+        description='Whether to use composed bringup',
+    )
+
+    declare_use_respawn_cmd = DeclareLaunchArgument(
+        'use_respawn',
+        default_value='False',
+        description='Whether to respawn if a node crashes. Applied when composition is disabled.',
+    )
+
+    declare_rviz_config_file_cmd = DeclareLaunchArgument(
+        'rviz_config_file',
+        default_value=os.path.join(
+            andino_navigation_dir, 'rviz', 'nav2_default_view.rviz'
+        ),
+        description='Full path to the RVIZ config file to use',
+    )
+
+    declare_use_simulator_cmd = DeclareLaunchArgument(
+        'use_simulator',
+        default_value='True',
+        description='Whether to start the simulator',
+    )
+
+    declare_use_rviz_cmd = DeclareLaunchArgument(
+        'use_rviz', default_value='True', description='Whether to start RVIZ'
+    )
+
+    declare_simulator_cmd = DeclareLaunchArgument(
+        'headless', default_value='False', description='Whether to execute gzclient)'
+    )
+
+    declare_world_cmd = DeclareLaunchArgument(
+        'world',
+        default_value=os.path.join(bringup_dir, 'worlds', 'world_only.model'),
+        description='Full path to world model file to load',
+    )
+
+    # Specify the actions
+    start_gazebo_server_cmd = ExecuteProcess(
+        condition=IfCondition(use_simulator),
+        cmd=[
+            'gzserver',
+            '-s',
+            'libgazebo_ros_init.so',
+            '-s',
+            'libgazebo_ros_factory.so',
+            world,
+        ],
+        cwd=[andino_navigation_dir],
+        output='screen',
+    )
+
+    start_gazebo_client_cmd = ExecuteProcess(
+        condition=IfCondition(PythonExpression([use_simulator, ' and not ', headless])),
+        cmd=['gzclient'],
+        cwd=[andino_navigation_dir],
+        output='screen',
+    )
```

Comment:
Why not using gazebo.launch.py like in andino_gazebo? Actually, why not calling that launch file?